imports/ui/pages/components/UserManagement/SecurityTab.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from "react";
+import { Button, Card, Input, notification } from "antd";
+
+import { translate } from "../../../HOCs/translate";
+import {
+  CURRENT_PASSWORD_PROPERTY,
+  CONFIRM_PASSWORD_PROPERTY,
+  NEW_PASSWORD_PROPERTY
+} from "../../../../constants/systemConstants";
+import { compose } from "redux";
+import { withTracker } from "meteor/react-meteor-data";
+import { mongoCss } from "../../../../api/client/collections";
+import injectSheet from "react-jss";
+import { dynamicUserProfileStyles } from "./dynamicUserProfileStyles";
+
+class SecurityTab extends Component {
```

Comment:
Looks like this is based on `Card` component and represent Card. Let's rename it to `SecurityCard`.